dspace-api/src/main/java/org/dspace/storage/rdbms/DatabaseUtils.java

```diff
@@ -328,24 +330,54 @@ public static void main(String[] argv) {
                     e.printStackTrace();
                     System.exit(1);
                 }
+            } else if (argv[0].equalsIgnoreCase("update-sequences")) {
+                try (Connection connection = dataSource.getConnection()) {
+                    String dbType = getDbType(connection);
+                    String sqlfile = "org/dspace/storage/rdbms/sqlmigration/" + dbType +
```

Comment:
Should this not be either absolute "/org/dspace/storage/rdbms/sqlmigration/" or relative "sqlmigration/"?

Reply:
The other worked, but it seems more clear to start with /.  Added commit e421a7c7c75

Reply:
The resource is not found if it is prefixed with /.  Perhaps that is treated as a filesystem path.  Reverting to "org/..."